extensions/visualizations/frequency_table_directive.html

```diff
@@ -29,4 +29,19 @@
     width: 60%;
     word-wrap: break-word;
   }
+  oppia-visualization-frequency-table table td:last-child {
```

Comment:
nit: combine this with the `nth-child` selector.

Reply:
Done!